app/_src/guides/federate.md

```diff
@@ -137,6 +138,33 @@ spec:
         maxRetries: 2
         maxRequests: 2" | kubectl --context=kind-mesh-global apply -f -
 ```
+{% endif_version %}
+{% if_version gte:2.9.x %}
+```
+echo "apiVersion: kuma.io/v1alpha1
+kind: MeshCircuitBreaker
+metadata:
+  name: demo-app-to-redis
+  namespace: {{site.mesh_namespace}}
+  labels:
+    kuma.io/mesh: default
+spec:
+  targetRef:
+    kind: MeshService
```

Comment:
this looks like a mistake, we can't use `kind: MeshService` with `name` and `namespace` in the top-level targetRef

Reply:
It's still in draft, let me look into that.